fall18/p4/README.md

```diff
@@ -1,3 +1,301 @@
 # Project 4
 
-More details coming soon.
+The project focuses on the IMDB movies dataset that contains a list of movies along with information like cast, director, rating, genre, plot description etc. You can open the IMDB-Movie-Data file in Excel or some other software to get an idea of what the dataset contains.
+
+We will be working with **strings**, **loops**, **conditionals** and **command line arguments** in this project.
+
+To start, download the 4 files given below into your project directory.
+
+* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/main.py)
+* [project.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/project.py)
+* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/test.py)
+* [IMDB-Movie-Data.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/IMDB-Movie-Data.csv)
+
```

Comment:
Open up the IMDB-Movie-Data.csv file and get familiar with it! Notice how the movies are indexed starting from zero, how the 'genres' and 'cast' are listed as several actors separated by spaces. There's something strange about the column 'budget', can you tell what it is?